docs/about/getting_started.md

```diff
@@ -19,6 +19,46 @@ To do so, clone the `ExpediaDotCom/haystack` repository and run the installer sc
 
 ### Install pre-requisites
```

Comment:
Use a blurb like : 

There are two possible approaches to getting Haystack running locally. Using `docker-compose` or using `minikube with terraform`

### Using docker-compose

#### Install pre-requisites